{
    "topic": "TS API Data Fetcher Bot Error",
    "pageNumber": "2",
    "type": "TS API Data Fetcher Bot Error - Reference Parent Missing",
    "definition": {
        "text": "The node where this error occurred needs to have a Reference Parent. Establish a reference from this node and this error will not happen again.",
        "updated": 1621860822884,
        "translations": [
            {
                "language": "TR",
                "text": "Özet: Bu hatanın oluştuğu düğümün bir Referans Üst Öğesine ( Reference Parent ) sahip olması gerekir. Bu düğümden bir referans oluşturun, bu hata tekrar olmayacak.",
                "updated": 1647598415454
            },
            {
                "language": "DE",
                "text": "Der Knoten, bei dem dieser Fehler auftrat, muß einen Referenz-Elternteil haben. Stellen Sie einen Verweis von diesem Knoten aus her und dieser Fehler wird nicht mehr auftreten. ",
                "updated": 1679246188020
            }
        ]
    },
    "paragraphs": [
        {
            "style": "Error",
            "text": "Reference Parent Missing",
            "updated": 1621860374194,
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata: Referans Üst Öğesi ( Reference Parent ) Eksik",
                    "updated": 1647598402086
                },
                {
                    "language": "DE",
                    "text": "Referenz-Elternteil (Reference Parent) fehlt",
                    "updated": 1679223680539
                }
            ]
        },
        {
            "style": "Title",
            "text": "Error Info",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Bilgisi",
                    "updated": 1647600675527
                },
                {
                    "language": "DE",
                    "text": "Fehlermeldung",
                    "updated": 1679223741857
                }
            ]
        },
        {
            "style": "Text",
            "text": "In the following section, you will find the information recorded when the error occurred.",
            "translations": [
                {
                    "language": "TR",
                    "text": "Aşağıdaki bölümde, hata oluştuğunda kaydedilen bilgileri bulacaksınız.",
                    "updated": 1647598434552
                },
                {
                    "language": "DE",
                    "text": "Im folgenden Abschnitt finden Sie die Informationen, die beim Auftreten des Fehlers aufgezeichnet wurden.",
                    "updated": 1679223710804
                }
            ]
        },
        {
            "style": "Text",
            "text": "You will only see the information if you opened this page as a result of the error actually occurring.",
            "updated": 1611680880668,
            "translations": [
                {
                    "language": "TR",
                    "text": "Bilgileri ancak bu sayfayı gerçekten oluşan hatanın bir sonucu olarak açtıysanız göreceksiniz.",
                    "updated": 1647598442460
                },
                {
                    "language": "DE",
                    "text": "Die Informationen werden nur angezeigt, wenn Sie diese Seite geöffnet haben und der Fehler tatsächlich aufgetreten ist.",
                    "updated": 1679223734514
                }
            ]
        },
        {
            "style": "Subtitle",
            "text": "Error Message",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Mesajı",
                    "updated": 1647598449379
                },
                {
                    "language": "DE",
                    "text": "Fehlermeldung",
                    "updated": 1679223767046
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Message of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata Mesajı aşağıdadır:",
                    "updated": 1647598507582
                },
                {
                    "language": "DE",
                    "text": "Im Folgenden wird die Fehlermeldung der Ausnahme angezeigt, die beim API Data Fetcher Bot aufgetreten ist:",
                    "updated": 1679223779552
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorMessage"
        },
        {
            "style": "Subtitle",
            "text": "Error Stack",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Yığını",
                    "updated": 1647598459518
                },
                {
                    "language": "DE",
                    "text": "Fehlerstapel",
                    "updated": 1679223787915
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Stack of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata Yığını aşağıdadır:",
                    "updated": 1647598515769
                },
                {
                    "language": "DE",
                    "text": "Nachfolgend ist der Fehlerstapel der Ausnahme aufgeführt, die beim API Data Fetcher Bot aufgetreten ist:",
                    "updated": 1679223805931
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorStack"
        },
        {
            "style": "Subtitle",
            "text": "Error Code",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Kodu",
                    "updated": 1647598466651
                },
                {
                    "language": "DE",
                    "text": "Fehlercode",
                    "updated": 1679223825013
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Code of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata Kodu aşağıdadır:",
                    "updated": 1647598535278
                },
                {
                    "language": "DE",
                    "text": "Im Folgenden ist der Fehlercode der Ausnahme aufgeführt, die beim API Data Fetcher Bot aufgetreten ist:",
                    "updated": 1679223836307
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorCode"
        },
        {
            "style": "Subtitle",
            "text": "Error Details",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Detayları",
                    "updated": 1647598475740
                },
                {
                    "language": "DE",
                    "text": "Fehler-Details",
                    "updated": 1679223847233
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following are all the Error details of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata detayları aşağıdadır:",
                    "updated": 1647598548005
                },
                {
                    "language": "DE",
                    "text": "Im Folgenden werden alle Fehlerdetails der Ausnahme aufgeführt, die beim API Data Fetcher Bot aufgetreten ist:",
                    "updated": 1679223875460
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorDetails"
        },
        {
            "style": "Subtitle",
            "text": "Node Name",
            "translations": [
                {
                    "language": "TR",
                    "text": "Düğüm Adı",
                    "updated": 1647598483217
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the node's name of the Node most related to this error:",
            "translations": [
                {
                    "language": "TR",
                    "text": "Aşağıdaki, bu hatayla en çok ilgili olan Düğümün ( Node ) adıdır:",
                    "updated": 1647598582564
                },
                {
                    "language": "DE",
                    "text": "Im Folgenden ist der Name des Knotens (Node) aufgeführt, der am meisten mit diesem Fehler in Verbindung steht:",
                    "updated": 1679223914921
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "nodeName"
        },
        {
            "style": "Subtitle",
            "text": "Node Code",
            "translations": [
                {
                    "language": "TR",
                    "text": "Düğüm Kodu",
                    "updated": 1647598490852
                },
                {
                    "language": "DE",
                    "text": "Knoten Code",
                    "updated": 1679223928656
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the code stored at the node that produced the error at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcısı Botunda ( API Data Fetcher Bot ) hatayı oluşturan düğümde depolanan kod aşağıda verilmiştir:",
                    "updated": 1647598592627
                },
                {
                    "language": "DE",
                    "text": "Der folgende Code ist auf dem Knoten gespeichert, der den Fehler im API Data Fetcher Bot verursacht hat:",
                    "updated": 1679223942397
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "nodeCode"
        }
    ]
}